lib/registries/addon/components/registries-search-result/template.hbs

```diff
@@ -0,0 +1,79 @@
+<div class="row">
+    <div class="col-xs-12">
+
+        <h4 local-class="RegistriesSearchResult__Title">
+            {{#if osfID}}
+                {{#link-to-external 'registration' osfID}}
+                    {{result.title}}
+                {{/link-to-external}}
+            {{else}}
+                <a href="{{result.mainLink}}">{{result.title}}</a>
+            {{/if}}
+
+            {{#if result.withdrawn}}
+                <span class="label label-default">{{t 'registries.discover.search_result.withdrawn'}}</span>
+            {{/if}}
+        </h4>
+
+        {{#comma-list result.contributors filter=contribFilter as |contrib|}}
+            {{contrib.users.citedAs}}
+        {{/comma-list}}
+
+        {{#if result.dateUpdated}}
+            <div class="m-t-sm">
+                <em> {{t 'registries.discover.search_result.last_edited'}}: {{moment-format (utc result.dateUpdated) 'MMMM D, YYYY UTC'}} </em>
```

Comment:
Can i18n handle date formatting?

Reply:
Oh, I guess not. I was thinking of how it can handle [pluralization](https://github.com/jamesarosen/ember-i18n/wiki/Doc:-Pluralization). There's [a ticket](https://openscience.atlassian.net/browse/EMB-249) for date formatting, so don't worry about it for now.